devito/nodes.py

```diff
@@ -341,7 +341,7 @@ def ccode(self):
         loop_body = [s.ccode for s in self.nodes]
 
         # Disregard offsets for buffered dimensions
-        offsets = [0, 0] if self.dim.is_Buffered else self.offsets
+        offsets = self.offsets
```

Comment:
You could just drop this line entirely (and the now outdated comment) and replace all occurrences of `offsets` with `self.offsets` in the rest of the function.